SJH/Day5/제일작은수제거하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(arr) {
+    if (arr.length === 1) {
+        return [-1];
+    }
+    const small = Math.min (...arr);
+    const smalldex = arr.indexOf (small);
+    arr.splice (smalldex, 1);
+    if (arr.length === 0) {
+        return [-1];
+    } else {
+        return arr;
+    }
```

Comment:
```suggestion
        return arr;
```
이미 위에서 `arr.length === 1`일 때`[-1]`을 반환하도록 처리해서, 그래서조건문 없이 `return arr;`써도 동작합니다.